wallet/wallet.go

```diff
@@ -1041,7 +1047,7 @@ func (w *Wallet) ImportPrivateKey(wif *btcutil.WIF, bs *waddrmgr.BlockStamp,
 
 // ExportWatchingWallet returns a watching-only version of the wallet serialized
 // in a map.
-func (w *Wallet) ExportWatchingWallet() (map[string]string, error) {
+func (w *Wallet) ExportWatchingWallet(walletPass string) (map[string]string, error) {
```

Comment:
The public passphrase should be reachable via the config, through the wallet receiver.  No need to pass it as a parameter.